README.md

```diff
@@ -76,6 +76,7 @@ Other open source projects that use Argo:
 * [applicationset-progressive-sync](https://github.com/Skyscanner/applicationset-progressive-sync) is a controller to allow a progressive sync of ArgoCD Applications generated by an ApplicationSet.
 * [argo-cd-action](https://github.com/clowdhaus/argo-cd-action) is a GitHub Action for executing Argo CD.
 * [argo-compare](https://github.com/shini4i/argo-compare) is a small CLI tool showing the difference in the resulting helm manifests rendered from Applications in different git branches.
+* [argo-cd-mixin](https://github.com/adinhodovic/argo-cd-mixin) is a monitoring mixin for ArgoCD. A set of Grafana dashboards and Prometheus rules for ArgoCD.
```

Comment:
Looks like it uses jsonnet though which is deprecated

Reply:
Wdym? Jsonnet is the primary language for generating grafana dashboards and prometheus alerts programatically. Similar projects are https://github.com/kubernetes-monitoring/kubernetes-mixin